### DISCUSSION TITLE
그룹#2 회고

### DISCUSSION THREAD

**Discussion by godrm** [codesquad-members-2022/swift-photoframe#137] [Show and tell]
여기에 각자 이번주 회고를 기록해주세요.
- 학습하면서 잘했던 점, 좋았던 점은 무엇인가요? (다음주에도 계속 유지해보세요)
- 학습하면서 스스로 아쉬웠던 점, 부족했던 점은 무엇인가요 - 그래서 어떻게 개선할 수 있을까요? (다음주에는 개선해봐요)
- 새롭게 알게 되거나 시도하지 못했지만, 다음주에 도전해볼건 무엇인가요? (다음주에 시도해봐요)
- 학습하는 데 조금이라도 도움이 된 사람이 있다면 칭찬해주세요!

**Comment by junu0516** [comment]
- 모르는 키워드가 나올때마다 최대한 공식문서를 보는 습관을 들이기 위해 노력했습니다. 한글이 아니다보니 아직 많이 낯설지만 다른 출처보다 결국 제일 정확한 설명이기 때문에 이해만 하면 해당 키워드를 확실히 알 수 있어 좋은 것 같습니다.

- 과제 외적으로 cs 공부 및 swift 학습이 많이 부진했는데 병행해서 같이 할 수 있도록 노력해야할 것 같습니다.(게을러지지 말자)

- cs10 과제 해결이나 알고리즘 문제를 풀 때는 옵셔널을 __`!`__ 로 강제로 벗겨냈지만 이번에는 좀 더 제대로 해보려고 If-let, guard-let을 사용하는 시도를 했습니다. 앞으로도 이런 식으로 권장되는 코드를 사용하는 습관을 들이고자 합니다.

- 모든 팀원들이 많은 도움을 주었습니다. 모르는 부분을 물어봐서 직접 도움을 주신 분들도 있었고, 같이 고민하면서 생각하지 못했던 부분을 일깨워주는 분도 있었습니다. 무엇보다 6시 이후에도 끊임 없이 오는 Pull Request 를 보면서 스스로 나태해지지 않게 동기부여가 많이 되었습니다.

**Reply by sungju-kim** [reply]
공식문서보다가 모르겠으면 제드한테 질문하면 되죠?

**Reply by junu0516** [reply]
ㅋㅋㅋㅋ ㅋㅋㅋㅋ 첫 4줄까지는 커버가능해요

**Reply by sungju-kim** [reply]
> ㅋㅋㅋㅋ ㅋㅋㅋㅋ 첫 4줄까지는 커버가능해요

다음주에는 8줄로 늘리시는걸로 ^^

**Reply by bibi6666667** [reply]
if let, guard let 훔쳐들으며 학습에 도움 많이 되었습니다👍

**Reply by BumgeunSong** [reply]
인터넷에 제드치니까 나옴 
![](https://i.ytimg.com/vi/V92wnlcymGM/maxresdefault.jpg)

**Reply by LeeJoobang** [reply]
> 공식문서보다가 모르겠으면 제드한테 질문하면 되죠?

데일과 함께 제드에게 질문하겠습니닿ㅎㅎㅎㅎ

**Reply by Jeeehee** [reply]
저도 제드 찾아갈게여

**Comment by sungju-kim** [comment]
- 학습하면서 잘했던 점, 좋았던 점은 무엇인가요? (다음주에도 계속 유지해보세요)
  - 학습을 하면서 태어나서 처음으로 정리와 TIL이란걸 작성해 보았습니다. 머리속에서 그리기만 하는게 아니라 글로 써내기 위해 자료를 좀더 꼼꼼히 보며 공식문서 보는법에 대해서 조금더 알게되었습니다.
  - TIL 과 별개로 공부해야할 키워드에 대해 Issue에 작성해 놓으니 다음에 무엇을 공부해야될지 확인하기 편했습니다.
- 학습하면서 스스로 아쉬웠던 점, 부족했던 점은 무엇인가요 - 그래서 어떻게 개선할 수 있을까요? (다음주에는 개선해봐요)
  - 미션 구현을 위한 학습이아닌 미션을 진행하며 옆으로 뻗어가는 학습이 필요한데, 어느부분에서 어떻게 뻗어가야할지 아직 길이 안잡힙니다.
  - 미션 구현을 위한 학습을 진행하며 새로 나오는 키워드에 대해 학습을 진행해 보고 있습니다. 잔가지가 부족한 느낌이지만 천천히 꾸준히 하다보면 가지수가 많아질거라 믿습니다.
- 새롭게 알게 되거나 시도하지 못했지만, 다음주에 도전해볼건 무엇인가요? (다음주에 시도해봐요)
  - Layout margin과 asset 에 대해 좀더 자세히 공부해보고 싶습니다.
  - 앱을 구현하면서 위 두가지 키워드는 빠질수 없는 키워드라고 생각해서 최소한 layout margin 에 대해서 느낌이라도 잡아보려고 합니다.
- 학습하는 데 조금이라도 도움이 된 사람이 있다면 칭찬해주세요!
  - 이번주의 가장 큰 벽.. git 이란 벽을 부수는데 많은 도움을 준 싱하, 구찌, 제드, 메이스 고마워요😁

**Reply by bibi6666667** [reply]
알고만 있는거랑 글로 적는거랑은 정말 다른 것 같아요.. 함께 TIL 화이팅입니다💪🏻

**Reply by Damagucci-Juice** [reply]
[sungju-kim/TIL](https://github.com/sungju-kim/TIL) 스타해주세요

**Reply by BumgeunSong** [reply]
데일이 TIL을 시작했다 이제 막을 수 없어 🔥

**Reply by sungju-kim** [reply]
> [sungju-kim/TIL](https://github.com/sungju-kim/TIL) 스타해주세요

스타 금지 입니다 (단호)

**Reply by LeeJoobang** [reply]
TIL까지 장착!!!!!!!!!!!!!!!!!

**Comment by JasonLee0223** [comment]
- 학습하면서 잘했던 점, 좋았던 점은 무엇인가요? (다음주에도 계속 유지해보세요)
↳ 다른 팀원들이 미션을 해나가는 모습을 보면서 많이 조급해했지만 Git 명령과 이해가 없으면 앞으로도
계속 힘들 것이라고 파악이되어 이번에 짚고 넘어갔는데 3일동안 고생했지만 내 흐름대로 밀고 나가서
한 주의 마지막에는 많이 이해하고 작업하는데 이상이 없어져서 앞으로도 나만의 흐름을 만드는 노력을
잘해야겠습니다.

- 학습하면서 스스로 아쉬웠던 점, 부족했던 점은 무엇인가요 - 그래서 어떻게 개선할 수 있을까요? (다음주에는 개선해봐요)
↳ 학습을 하는데 있어 한 번 이해가 안되면 다 이해할 때까지 계속 학습하는 성향이 안바뀌어서 이후에 급하게
진행할 때 불편할 것 같아 조금은 고쳐보도록 해야할 것 같습니다. (다 이해하려고 하지말고 해당 상황을 모면할 수 있게끔 이해하고 넘어가도록 불편한 마음을 비우는 일을 해봐야할 것 같습니다.)

- 새롭게 알게 되거나 시도하지 못했지만, 다음주에 도전해볼건 무엇인가요? (다음주에 시도해봐요)
↳ Eddy가 알려준 영문내용들을 앞으로 읽을 일이 많을텐데 시간을 갖고 스플릿뷰로 읽으면서 글을 써보도록 시도해봐야겠다!

- 학습하는 데 조금이라도 도움이 된 사람이 있다면 칭찬해주세요!
↳ 모든 팀원들이 자기일처럼 도와주셔서 감사했습니다! 다른 분들이 PR을 보내는것을 보고 더 열심히 하게되는점도 좋았습니다! ㅎㅎ 이번에 Git 문제점 해결을 도와주신 구찌, 싱하 너무 감사드립니다!🙇🏻

**Reply by sungju-kim** [reply]
저와는 완전 반대되는 학습 성향을 가지고 계시네요 ㅋㅋㅋㅋㅋㅋ 반만 주세요,,

**Reply by JasonLee0223** [reply]
저는 데일의 학습 성향이 부러운데 ㅋㅋㅋㅋㅋ 반씩 가져가면 완벽하겠군요!🤓

**Reply by bibi6666667** [reply]
PR로 공부자극 받는 거 공감입니다ㅎㅎ 제이슨은 항상 열공하시니까 깃도 개발도 금방 느실 거에요🎉

**Reply by Damagucci-Juice** [reply]
열공러 제이슨 야야야

**Reply by LeeJoobang** [reply]
제이슨 우리 손잡고 화이팅:raised_hands::raised_hands::raised_hands::raised_hands::raised_hands::raised_hands::raised_hands::raised_hands:

**Comment by shingha1124** [comment]
* **학습하면서 잘했던 점, 좋았던 점은 무엇인가요? (다음주에도 계속 유지해보세요)**
  * NavigationController와 TabbarController와 같은 컨트롤러들을 어떻게 동작하는지 모르고 사용하곤 했었는데
    이번 과정을 통해 컨트롤러 전환이나, 동작 방법에 대해 알게 되었습니다.
* **학습하면서 스스로 아쉬웠던 점, 부족했던 점은 무엇인가요 - 그래서 어떻게 개선할 수 있을까요? (다음주에는 개선해봐요)**
  * 익숙한거로만 계속 사용해 보려는 습관이 아직 남아 있는거같아서 아쉬웠습니다. 
  * 새로운 시도를 많이 해보지 못한거 같습니다
  * git 역시 평소와 쓰던 대로만 사용해서, 몇번의 실수가 생겼었습니다.
  * 코드를 짜면서 새로운 방법들이 있는지 찾아보고, 여러가지 방면으로 테스트를 진행해 보겠습니다
* **새롭게 알게 되거나 시도하지 못했지만, 다음주에 도전해볼건 무엇인가요? (다음주에 시도해봐요)**
  * NavigationViewController, TabBarViewController, TableViewController등 다양한 컨테이너들의 사용 확인
  * AppDelegate, SceneDelegate 등 앱의 시작을 관리하는 것들을 알아보고자 합니다.
* **학습하는 데 조금이라도 도움이 된 사람이 있다면 칭찬해주세요!**
  * 모든 분들의 PR을 보면서 많은 도움이 되었습니다.
  * 새로운 방법으로 진행해 보신 분들을 보면서, 참조해서 테스트 해보고, 여러가지 방법의 코딩을 알게되었습니다.

**Reply by junu0516** [reply]
깃 명강의 감사합니다....!!

**Reply by BumgeunSong** [reply]
싱하 덕분에 git과 싸우는데 많은 도움이 되었습니다!! 👍

**Reply by Damagucci-Juice** [reply]
싱하 쾌차 기원 3일차

**Reply by bibi6666667** [reply]
저도 깃 디버깅에 많은 도움 주셔서 감사합니다 싱하!!
나중에 rebase 관련 특강 하시면 꼭 슬랙에 알려주세요ㅎㅎ (1/28)
자가격리 화이팅하시고 얼른 쾌차하시길..!💊

**Reply by sungju-kim** [reply]
깃 ... 싱하 덕분에 두렵지 않습니다  😀

**Reply by JasonLee0223** [reply]
항상 도와주셔서 감사합니다!!🙇🏻 얼른 쾌차하셔서 활동하실 수 있으면 좋겠어요! ㅎㅎ

**Reply by LeeJoobang** [reply]
싱하 아프지마.....싱하 못잃어ㅜㅜㅜㅜㅜㅜㅜㅜㅜ

**Reply by BumgeunSong** [reply]
싱하 만수무강 기원

![](https://ogq-sticker-global-cdn-z01.afreecatv.com/sticker/176b146bed60b68/2.png)

**Reply by Jeeehee** [reply]
외쳐 갓싱하! 아프지마오ㅠ

**Reply by piggyse** [reply]
언제나 든든한 싱하 너무 감사합니다!

**Reply by Jinsujin** [reply]
소스트리 마스터 싱하...우리 몸 건강부터 챙기면서 해요...😭

**Comment by Damagucci-Juice** [comment]
* 좋았던 점 
* * todolist 에서 todo schedule로 변경하니 예상 작업시간이라는 개념이 생겨서 좋았습니다.
* - 아는 지식을 사람들고 나누니 그룹에 도움된다는 생각에 뿌듯했습니다.

* 아쉬웠던 점
* * 무리한 학습계획 : 너무 많은 todo를 세워서 스스로를 몰아 세워서 힘들었습니다.
* * 쉴 때 너무 푹쉬는 느낌 : 계획적으로 쉬고 싶었는데 아쉬었습니다.

* 해볼것
* * 세그웨이 없이 다음 뷰 컨트롤러를 보기
* * 꾸준 독서, 꾸준 영어 독해, 꾸준 블로깅 : 또 너무 많은 할일을 만든건아닌지??ㅋㅋㅋㅋ
* * Container root View cantroller와 master/detail view Controller를 만들어서 split view controller 환경을 만들어보기(아이패드용)

* 칭찬해요!
* * 싱하 : 지식과 겸손 두마리를 잡은 무서운 분입니다..
* * 에디 : 묘하게 듣게 되는 마성의 남자
* * 데일 : 열정 만수르 1
* * 비비 : 크림...하하하 농담이고 비비 너무 열정만수르 2
* * 제드 : 제드랑 이야기하면 그냥 힐링이 됩니다.
* * 제이슨 : 배울 때 꾸준히 성실한 자세가 멋집니다.

**Reply by BumgeunSong** [reply]
비비크림.. 아웃 🖐

**Reply by Damagucci-Juice** [reply]
ㅋㅋ;;

**Reply by bibi6666667** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋ ㅁ..멈춰! 🖐

**Reply by sungju-kim** [reply]
ㅋㅋㅋㅋ비비크림ㅋㅋㅋㅋㅋㅋ전 재밌는데 왜죠 ㅋㅋㅋㅋㅋ

**Reply by JasonLee0223** [reply]
ㅋㅋㅋㅋㅋㅋㅋ 비비크림...🤪 오늘 호눅스 책이 도착했네요! ㅎㅎ 추천감사해요 주말에 한 번 읽어보도록 노력해볼께요👊🏻

**Reply by LeeJoobang** [reply]
구찌 아니었음 평생 모니터 살 생각도 못했을거예요....ㅜㅜㅜㅜ

**Reply by BumgeunSong** [reply]
![](https://jjalbang.today/files/jjalbox/2015/12/20151202_565e74ef985ee.jpg)

**Comment by bibi6666667** [comment]
- 학습하면서 잘했던 점, 좋았던 점은 무엇인가요? (다음주에도 계속 유지해보세요)
  - 좋지 못한 컨디션 + 정신없는 환경 에도 가능한 꾸준히 공부한 것?
  - 하던 운동을 꾸준히 계속한 것

- 학습하면서 스스로 아쉬웠던 점, 부족했던 점은 무엇인가요 - 그래서 어떻게 개선할 수 있을까요? (다음주에는 개선해봐요)
  - 전체적으로 학습 중 집중력이 낮았던 것. 공부시간을 알차게 활용하지 못한 것.
    - 다음 주에는 공부 시간에 더 집중함으로써 수면시간을 보장하는 선순환 구조를 만들기 위해 노력하겠습니다..!!
  - 학습 정리하고 진도 나가는 데 치중하느라 질문을 많이 하지 못한 것
    - 의문이 드는 점은 바로 질문하도록 노력해 보겠습니다

- 새롭게 알게 되거나 시도하지 못했지만, 다음주에 도전해볼건 무엇인가요? (다음주에 시도해봐요)
  - 주말 동안 나머지 미션 진도를 다 나가 보고, learngitbranching 학습도 해보려 합니다.

- 학습하는 데 조금이라도 도움이 된 사람이 있다면 칭찬해주세요!
  - 모든 팀원들이 감사했습니다!
  - 그룹2의 유재석 에디
  - 다이나믹한 표정을 보여주신 다마구찌
  - 마음이 평온해지는 목소리를 가지신 제드
  - 깃 디버깅에 많은 도움을 주신 갓-싱하
  - 귀여운 메밀🐱이를 보여주신 데일
  - 맥미니를 장착해 더욱 강해진 열공러 제이슨

**Reply by BumgeunSong** [reply]
요가 + 알바 + 이사 + 코딩 = 갓생 비비

**Reply by Damagucci-Juice** [reply]
마스터/비비 
- 알바/비비 
- 요가/비비
- 이사/비비
- 코딩/비비

**Reply by sungju-kim** [reply]
> 마스터/비비
> 
> * 알바/비비
> * 요가/비비
> * 이사/비비
> * 코딩/비비

git reset --hard

**Reply by JasonLee0223** [reply]
↳ 222222 ㅋㅋㅋㅋㅋㅋ 비비 쉬엄쉬엄해요!! 😎

**Reply by sungju-kim** [reply]
비비의 하루는 48시간 맞죠 ?

**Reply by Jeeehee** [reply]
세심한 비비..

**Comment by BumgeunSong** [comment]
**(+) 학습하면서 잘했던 점, 좋았던 점은 무엇인가요? (다음주에도 계속 유지해보세요)**
- 주어진 과제에서 벗어나 **이것저것 뿌셔보고 바꿔보면서** 배웠습니다. 이렇게 건드려보다보니 다양한 컨트롤러와 UI 요소들를 다루는 것이 (익숙하진 않아도) 적어도 친숙해지는 느낌이었습니다. 
- **공식 문서**를 꼼꼼히 읽어보았습니다. 애플 공식 문서에 애정이 잘 가지 않았는데, ViewController와 관련된 부분들은 워낙 중요하다보니 친절하게 잘 쓰여있어서 공부하기 좋았습니다.
- **git**을 다루는 게 조금 더 익숙해져서 좋았습니다! 괴로웠지만 생각해보니 1주일만에 많은 진화를 한듯 합니다

**(-) 학습하면서 스스로 아쉬웠던 점, 부족했던 점은 무엇인가요 - 그래서 어떻게 개선할 수 있을까요? (다음주에는 개선해봐요)**
- git의 원리와 실습에 대해 아직 좀 부족하다고 느끼고 있습니다. Git command에 아직 자신감이 없네요 😮‍💨 
- 어제 산 **호눅스의 Git 책, 깃 실습 사이트 미션**을 더 시도해봐야겠습니다.

**(*) 새롭게 알게 되거나 시도하지 못했지만, 다음주에 도전해볼건 무엇인가요? (다음주에 시도해봐요)**
- 다음주에는 git의 원리/실습한 내용을 바탕으로 **개인 블로그 포스팅**을 해볼 생각입니다. ✍️

**(/) 학습하는 데 조금이라도 도움이 된 사람이 있다면 칭찬해주세요!**
- 도움을 주는 분들도 다 훌륭했지만, **Gucci** / **Dale** 이 항상 리액션을 잘해주고, 적극적으로 도움 요청을 하면서 서로 도와주는 '분위기'를 만들어줘서 칭찬하고 싶네요 👍

**Reply by Damagucci-Juice** [reply]
좋은 책 추천해주셔서 고마워요 딥워크 이제 배송 왔습니다 ㅎㅎ

**Reply by bibi6666667** [reply]
공식문서.. 정말 애증의 관계인 것 같네요🙃

**Reply by sungju-kim** [reply]
블로그 포스팅 기다리고 있겠습니다 😄

**Reply by TaeKyeongKim** [reply]
공식 문서 ViewController ... 메모..

**Reply by JasonLee0223** [reply]
Eddy! 한 주 오디오가 비면 주방처럼 매워주시고 좋은 학습방법 공유해주셔서 감사합니다!! 👍🏻👍🏻

**Reply by Jeeehee** [reply]
에디 블로그 읽으먄서 겅부하고있슴다..

**Reply by LeeJoobang** [reply]
블로그 기다립니다:pray:

**Reply by Jinsujin** [reply]
기다립니다2222 🙏
